packages/console/src/components/common/utils.ts

```diff
@@ -27,13 +27,15 @@ export function measureText(fontDefinition: string, text: string) {
  */
 export const checkForDynamicExecutions = (allExecutions, staticExecutions) => {
   const parentsToFetch = {};
+  const executionsByNodeId = {};
   for (const executionId in allExecutions) {
+    const execution = allExecutions[executionId];
```

Comment:
let's keep the `dynamicExecution` name, so we don't trigger the changes here, and `executionsByNodeId` can be transformed to `dynamicExecutionsByNodeId`

Reply:
@olga-union. As this execution can be dynamic or static, so I don't think we can name it as dynamicExecution.